packages/store/ts/config/v2/table.ts

```diff
@@ -126,7 +126,7 @@ export function resolveTableCodegen<input extends TableInput>(input: input): res
     storeArgument: get(options, "storeArgument") ?? TABLE_CODEGEN_DEFAULTS.storeArgument,
     // dataStruct is true if there are at least 2 value fields
     dataStruct: get(options, "dataStruct") ?? Object.keys(input.schema).length - input.key.length > 1,
-  } satisfies TableCodegen as resolveTableCodegen<input>;
+  } satisfies TableCodegen as never;
```

Comment:
🫨

Reply:
this just aligns with the rest of our `as never` usage, context here: https://discord.com/channels/865335009915961364/1199772289495990404/1230849460842532915

Reply:
very interesting, thank you!